lib/web/ui/cluster.go

```diff
@@ -126,5 +129,12 @@ func GetClusterDetails(ctx context.Context, site reversetunnelclient.RemoteSite,
 		// this code runs in the proxy service, so we can safely
 		// use the version embedded in the binary
 		ProxyVersion: teleport.Version,
-	}, nil
+	}
+
+	licenseExpiry := modules.GetModules().LicenseExpiry()
+	if !licenseExpiry.IsZero() {
+		clusterDetails.LicenseExpiry = licenseExpiry
+	}
```

Comment:
hrm i don't think this will work, b/c it's the auth server that reads the license and sets the modules there, the proxy can be ran separately so this can return zero value

Reply:
normally you'd call `ping` to get the latest flags

Reply:
Yeah, the auth server should check modules and stuff the result in the response. We can probably use whatever request this page already makes so we don't have to do a separate ping.